instrumentation/trilogy/lib/opentelemetry/instrumentation/trilogy.rb

```diff
@@ -11,6 +11,45 @@ module OpenTelemetry
   module Instrumentation
     # Contains the OpenTelemetry instrumentation for the Trilogy gem
     module Trilogy
+      extend self
+
+      CURRENT_ATTRIBUTES_KEY = Context.create_key('mysql-attributes-hash')
```

Comment:
```suggestion
      CURRENT_ATTRIBUTES_KEY = Context.create_key('trilogy-attributes-hash')
```

Reply:
@arielvalentin I thought the plan was to eventually extract this into a shared MySQL context helper 🤔 Shouldn't the key be kept the same in that case? (e.g. https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/348#discussion_r1120727262)

Reply:
I can change this to be `trilogy-attributes-hash`, but will this change to `mysql-attributes-hash` when the helper is eventually extracted?

Reply:
That will be OK then. I think since they are different Key Object Instances in this commit I would prefer them to have different names.

Reply:
Sounds good! Will update 👍

Reply:
Okay, https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/347 is updated with the change. Will rebase this PR once the aforementioned one ships!